includes/activity/class-base-object.php

```diff
@@ -692,4 +700,13 @@ public function to_json() {
 	public function get_object_var_keys() {
 		return \array_keys( \get_object_vars( $this ) );
 	}
+
+	/**
+	 * Returns the JSON-LD context of this object.
+	 *
+	 * @return array $context A compacted JSON-LD context for the ActivityPub object.
+	 */
+	public function get_json_ld_context() {
+		return $this::JSON_LD_CONTEXT;
```

Comment:
shouldn't this be `self::JSON_LD_CONTEXT`?
```suggestion
		return self::JSON_LD_CONTEXT;
```